ivy/data_classes/container/creation.py

```diff
@@ -1583,7 +1583,7 @@ def frombuffer(
         ...     a = b'\x00\x00\x00\x00\x00\x00\xf0?',
         ...     b = b'\x00\x00\x00\x00\x00\x00\xf0?\x00\x00\x00\x00\x00\x00\x00@'
         ... )
-        >>> y = ivy.Container.static_frombuffer(x)
+        >>> y = ivy.Container.frombuffer(x, dtype=ivy.float64)
```

Comment:
as this is instance method example, it should be `y = x.frombuffer(dtype=ivy.float64)`
I am not sure why the docstring test is missing this error out when I ran it locally from this branch. :thinking:

Reply:
docstring test can't check if container or array method used correctly. I tried to correct as long as I noticed.